eras/conway/test-suite/test/Test/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -4,18 +4,12 @@
 
 module Test.Cardano.Ledger.Conway.TxInfo where
 
-import Cardano.Ledger.Alonzo.Plutus.TxInfo (ExtendedUTxO (..))
-import qualified Cardano.Ledger.Babbage.TxBody as B
-import Cardano.Ledger.Core hiding (TranslationError)
-import Cardano.Ledger.Plutus.Language (Language (..))
 import Data.Proxy (Proxy (..))
-import Test.Tasty (TestTree)
-
-import qualified Test.Cardano.Ledger.Babbage.TxInfo as B
+import Test.Tasty (TestTree, testGroup)
 
 txInfoTests ::
   forall era.
-  (ExtendedUTxO era, EraTx era, B.BabbageEraTxBody era, B.BabbageTxInfoTests era) =>
   Proxy era ->
   TestTree
-txInfoTests p = B.txInfoTestsV2 p PlutusV3 -- The V2 tests in Babbage should all hold for V3
+txInfoTests _p = testGroup "Conway Plutus Tests" [] -- Disabled for now
```

Comment:
Make sure to have an issue to re-enable this at some point.

Reply:
Done: https://github.com/input-output-hk/cardano-ledger/issues/3923